app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,10 @@
+# app/controllers/foods_controller.rb
+
+require 'faraday'
+
+class FoodsController < ApplicationController
+  def index
+    @total_hits = FoodFacade.total_hits(params[:query])
+    @foods = FoodFacade.foods_containing_ingredient(params[:query])
```

Comment:
The text field label in the search form is `q`, so we don't actually have a param called `query`. If you noticed your API call wasn't returning quite what you expected, it's because there was no query param being sent with the request. If we update this to `params[:q]` it should work!